sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -288,4 +288,56 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-32234: orc data created by the hive tables having _col fields" +
+    " name for vectorized reader") {
+    Seq(false, true).foreach { vectorized =>
+      withSQLConf(SQLConf.ORC_VECTORIZED_READER_ENABLED.key -> vectorized.toString) {
+        withTable("test_hive_orc_vect_read") {
+          spark.sql(
+            """
+              | CREATE TABLE test_hive_orc_vect_read
+              | (_col1 INT, _col2 STRING, _col3 INT)
+              | USING orc
+            """.stripMargin)
+          spark.sql(
+            """
+              | INSERT INTO
+              | test_hive_orc_vect_read
+              | VALUES(9, '12', 2020)
+            """.stripMargin)
+
+          val df = spark.sql("SELECT _col2 FROM test_hive_orc_vect_read")
+          checkAnswer(df, Row("12"))
+        }
+      }
+    }
+  }
+
+  test("SPARK-32234: orc data created by the hive tables having _col fields name" +
```

Comment:
Please merge two test cases like the following in order to have a full test coverage.

```scala
Seq("native", "hive").foreach { orcImpl =>
  Seq(false, true).foreach { vectorized =>
```

Reply:
Merge the test